django/thunderstore/repository/models/service_account.py

```diff
@@ -0,0 +1,18 @@
+import ulid2
+from django.conf import settings
+from django.db import models
+
+
+class ServiceAccountMetadata(models.Model):
```

Comment:
Maybe we could place this into a new app, e.g. `thunderstore.account`? It isn't technically related to the repository so feels like a weird place. Additionally, maybe we could just call this `ServiceAccount`, given that we will not be naming any other resource like that?

Reply:
I originally thought about `thunderstore.core` but because of the connection to `UploaderIdentity` I wasn't sure if that was enough to move it to `thunderstore.repository`.

For the naming, to me, `ServiceAccount` sounds like an alternative to `User`, maybe with more fields. The names in a few places could probably be improved but if we decide to call this extra information about the `User` `service_account` then that simplifies things. The name might also depend on https://github.com/risk-of-thunder/Thunderstore/pull/222#discussion_r559238642

Reply:
Replied on the linked comment